src/routes.ts

```diff
@@ -48,6 +48,10 @@ routers.get('/api/complaints/votes', async (req: Request, resp: Response) => {
 	return await complaintProxy.getComplaintWithVote(req, resp);
 });
 
+routers.delete('/api/complaint/delete', async (req: Request, resp: Response) => {
```

Comment:
Mudem a URL dessa rota pfvr, para /api/complaints. Para seguir o padrão REST de nomeação de rotas, deixamos as rotas de CRUD iguais mas com métodos diferentes, então vcs podem colocar dessa maneira e deixar o método como delete msm. Depois lembrem de arrumar no front tbm

Reply:
Obrigado pelo feedback, a rota foi colocada no padrão REST e mudada no nosso serviço de Front.

Reply:
Boa. Mas só outro feedback. Eu vi que mudou de /api/complaint/delete para /api/complaints/delete, mas o que eu queria dizer mesmo era que o padrão seria mudar de /api/complaint/delete para apenas /api/complaints. 

O padrão REST indica que as rotas devem ficar no plural e que esses substantivos (create, read, delete e parecidos) devem ficar subentendidos nos métodos que estão sendo chamados pelas controllers. Então eles não devem aparecer nas rotas. Se você ver, nas outras rotas, não tem mais /api/complaints/create ou /api/votes/add ou list. Esses métodos create e list estão sendo subentendidos nas controllers e não precisam ser chamados na url da rota. 

O Gui Marques sabe explicar melhor, mas o básico é isso. Se puderem revisar novamente

Reply:
Ah, acho que eu não comentei aqui, mas vejam também a questão dos métodos da outra issue de vocês estar aqui também. Se for igual, é melhor tirar desse PR

Reply:
Bom dia, avia entendido errado a correção, creio que agora está no modelo correto